specification/monitor/resource-manager/readme.go.md

```diff
@@ -130,4 +136,58 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag) == 'package-2021-09' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2021-09-01-preview/$(namespace)
-```
\ No newline at end of file
+```
+
+### Tag: package-2022-03-preview-datacollection and go
+
+These settings apply only when `--tag=package-2022-03-preview-datacollection --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-preview-datacollection' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2022-03-26-preview/$(namespace)
+```
+
+### Tag: package-2022-03-part5 and go
+
+These settings apply only when `--tag=package-2022-03-part5 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-part5' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2022-03-25/$(namespace)
+```
+
+### Tag: package-2022-03-part4 and go
+
+These settings apply only when `--tag=package-2022-03-part4 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-part4' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2022-03-24/$(namespace)
+```
+
+### Tag: package-2022-03-part3 and go
+
+These settings apply only when `--tag=package-2022-03-part3 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-part3' && $(go)
+output-folder: $(go-sdk-folder)/services/monitor/mgmt/2022-03-23/$(namespace)
+```
+
+### Tag: package-2022-03-preview-part2 and go
+
+These settings apply only when `--tag=package-2022-03-preview-part2 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-preview-part2' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2022-03-22-preview/$(namespace)
+```
+
+### Tag: package-2022-03-preview-part1 and go
+
+These settings apply only when `--tag=package-2022-03-preview-part1 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2022-03-preview-part1' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2022-03-21-preview/$(namespace)
```

Comment:
I run out of ideas for names. How about we just call this `monitorcore` or `monitorlegacy` or `legacymonitor`
```suggestion
output-folder: $(go-sdk-folder)/services/preview/monitor/mgmt/2019-03-01-preview/monitorlegacy
```